### PR TITLE
Buy albums

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

Buy albums

## Trello Card

https://trello.com/c/1UbUdZWz

### CODE REVIEW

File: app/graphql/albums/middlewares.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const errors = require('../../errors'),
```
Comment (gonziesc) [inline app/graphql/albums/middlewares.js]: it's interesting what you did. We see the concept of middleware as a reusable component between endpoints. I don't see that in this usecase. Anyways, you pre validation isolated from the resolver it's nice! I'd just move it to another place, like an interactor

Comment (YesicaNavaB) [reply app/graphql/albums/middlewares.js]: ok!

File: app/graphql/albums/resolvers.js [hunk 2]
```diff
@@ -13,11 +17,26 @@ const getAlbumsList = (_, params) =>
     return orderArrayByField(albumsPagitation, params.orderBy);
   });
 
+const buyAlbum = async (_, params, context) => {
+  try {
+    const data = await validatorBuyAlbums({ ...params, authorization: context.authorization });
+    await Album.createModel(data);
```
Comment (gonziesc) [inline app/graphql/albums/resolvers.js]: shouldnt we return this information?

File: test/albums/resolvers.spec.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const { Mutation: mutations } = require('../../app/graphql/albums/resolvers');
+
+describe('Albums', () => {
```
Comment (gonziesc) [inline test/albums/resolvers.spec.js:3]: what about failing cases?

Comment (YesicaNavaB) [reply test/albums/resolvers.spec.js:3]: ok!

File: app/graphql/albums/resolvers.js (hunk 2 above)
Comment (YesicaNavaB) [reply app/graphql/albums/resolvers.js]: ok!

**Review by gonziesc — APPROVED** [review]

**Pull request merged**